js/constants/appConfig.js

```diff
@@ -78,6 +79,9 @@ module.exports = {
     msBetweenRechecks: 1000 * 60 * 60 * 2, // 2 hours
     enabled: true
   },
+  safeBrowsingAll: {
+    enabled: true
```

Comment:
does this correspond to enabling both brave and google safebrowsing? shouldn't it default to false?

Reply:
safeBrowsingAll - Google + Brave Safe Browsing
safeBrowsing - Brave Safe Browsing

I was thinking `safeBrowsingAll` will be the `safeBrowsingControl` by default.

Reply:
i thought we decided safeBrowsing (without Google) would be the default, but cc @flamsmark

Reply:
I think we should use both by default. If it's not the default, I can't imagine how many people would turn it on. And this is a protection which most benefits those least likely to wrangle their settings.

Reply:
alright defaulting to both sgtm